lib/l10n/intl_en.arb

```
@@ -796,9 +796,22 @@
 
 
   "save": "Save",
+  "apply": "Apply",
   "selectCurrency": "Select Currency",
   "noCurrencySelected": "No currency selected",
 
+  "@_comment_filter_section": "Filter section strings",
+  "fiatCurrencies": "Fiat currencies",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)